library/src/main/java/org/apache/apex/malhar/lib/window/accumulation/PojoFullOuterJoin.java

```diff
@@ -44,6 +44,12 @@ public PojoFullOuterJoin(int num, Class<?> outClass, String... keys)
     super(outClass,keys);
   }
 
+  public PojoFullOuterJoin(Class<?> outClass, String[] leftKeys, String[] rightKeys)
```

Comment:
This is a correct ctor... Please remove the previous one i.e. public PojoFullOuterJoin(int num, Class<?> outClass, String... keys)... I'm not somehow not able to add comment on line no 42.